site/docs/v1/tech/tutorials/gating/registration-gate-accounts-until-verified.adoc

```diff
@@ -0,0 +1,188 @@
+
+image::tutorials/registration-gate-accounts-until-verified/registration-template-customization.png[Customizing the registration verification email template,width=1200]
+
+== Test It Out
+
+Now, test out the functionality. Register a new account. Make sure to use a different browser or incognito mode so that the admin account you logged in with while configuring FusionAuth doesn't interfere with new user registration.
+
+Visit the registration URL copied above. You'll be prompted to register. Fill out the form with a valid email address. After registration is complete, you'll receive an email prompting you to verify the registration.
+
+Here is the email in the mailcatcher user interface:
+
+image::tutorials/registration-gate-accounts-until-verified/verification-registration-email-mailcatcher.png[Verification registration in mailcatcher,width=1200,role=bottom-cropped]
+
+This user account will be gated and unable to completely log in if you have not verified the registration through email. In your browser where you registered, you'll see a page similar to below:
+
+image::tutorials/registration-gate-accounts-until-verified/registration-verification-gate-screen-clickable-link.png[Clickable link default registration verification gate screen,width=1200,role=bottom-cropped]
+
+To test the gating, open a different browser (not a window, but an entirely different browser) and visit the login URL copied above. Enter the email and password for the user you just registered.
+
+You'll be sent to the same registration verification gate page.
+
+No matter which API or hosted login page you use, no JWT will be issued until the user registration has been verified. The user will continue to see the verification prompt when they log in until then.
+
+If an API is being used to authenticate the user, the API will return a status code indicating registration verification has not yet occurred. Please see the link:/docs/v1/tech/apis/login/[Login API] for more details.
+
+If you view the user in the administrative user interface and click on the `source` tab you should see the following attributes on the user.
+
+image::tutorials/registration-gate-accounts-until-verified/unverified-user-registration-details-screen.png[Unverified user details screen,width=1200,role=bottom-cropped]
+
+Once you have verified the email address by clicking on the link, you can now log in and will be sent to the configured redirect URL.
+
+== The User Object
+
+The user object changes before, during, and after the registration verification occurs. Let's look at the JSON.
+
+If a user is created and registration verification is disabled, the verified attribute of the registration object will be true. This is because FusionAuth treats all users as having verified registration (via email) when registration verification is disabled.
+
+[source,json]
+.User Object With Registration Verification Turned Off Initially JSON
```

Comment:
This isn't really the user object at this point. It is the Registrations array, excerpted from the User object.

Please update the title of this JSON fragment.